sim/dex.ts

```diff
@@ -472,6 +480,35 @@ export class ModdedDex {
 		return learnsetData;
 	}
 
+	getDescs(table: keyof TextTableData, id: ID, dataEntry: AnyObject) {
+		if (dataEntry.shortDesc) {
```

Comment:
Why just `shortDesc`? It's confusing particularly for items since many of them only have a `desc`, so many OMs with custom items (e.g. SSB) only specify an updated `desc`. (Or should `shortDesc` be always present, and `desc` only for things that need it?)

Reply:
I've never actually standardized what it means for only one of `desc` or `shortDesc` to exist. I think most of our accesses backfill in both directions. If you'd like to standardize, please do, and document that in our TS types.

Reply:
Actually what I was actually concerned about is about how mods override descriptions. Hopefully none of them want to override just one of `desc` and `shortDesc`, so it should be enough to change this to test both.

Reply:
Ugh, it took me 20 minutes to compose that and I still made a fool of myself...